installer/roles/ansible-service-broker-setup/tasks/main.yml

```diff
@@ -52,6 +56,7 @@
   - dockerhub_org != ''
   - launch_apb_on_bind is defined
   - launch_apb_on_bind != ''
+  - "'ansible-service-broker' not in all_projects.stdout"
```

Comment:
Best to introduce variable for this as we hardcode this value in couple places.
We hardcode it: 
https://github.com/aerogear/mobile-core/blob/master/installer/roles/ansible-service-broker-setup/files/provision-ansible-service-broker.sh#L29